chacra/async/debian.py

```diff
@@ -66,7 +66,8 @@ def create_deb_repo(repo_id):
                 None,
                 repo.distro_version,
                 distro_versions=combined_versions,
-                ref=ref if ref != 'all' else None
+                ref=ref if ref != 'all' else None,
+                sha1=repo.sha1
```

Comment:
I could use the ``combine_deb_repos`` config option we already have to optionally apply this filter I guess. Thoughts?